src/bin/list_store.rs

```diff
@@ -0,0 +1,299 @@
+        },
+        Data {
+            fixed: true,
+            number: 52877,
+            severity: "Major".to_string(),
+            description: "GtkFileSelection needs a refresh method. ".to_string(),
+        },
+        Data {
+            fixed: false,
+            number: 56070,
+            severity: "Normal".to_string(),
+            description: "Can't click button after setting in sensitive".to_string(),
+        },
+        Data {
+            fixed: true,
+            number: 56355,
+            severity: "Normal".to_string(),
+            description: "GtkLabel - Not all changes propagate correctly".to_string(),
+        },
+        Data {
+            fixed: false,
+            number: 50055,
+            severity: "Normal".to_string(),
+            description: "Rework width/height computations for TreeView".to_string(),
+        },
+        Data {
+            fixed: false,
+            number: 58278,
+            severity: "Normal".to_string(),
+            description: "gtk_dialog_set_response_sensitive () doesn't work".to_string(),
+        },
+        Data {
+            fixed: false,
+            number: 55767,
+            severity: "Normal".to_string(),
+            description: "Getters for all setters".to_string(),
+        },
+        Data {
+            fixed: false,
+            number: 56925,
+            severity: "Normal".to_string(),
+            description: "Gtkcalender size".to_string(),
+        },
+        Data {
+            fixed: false,
+            number: 56221,
+            severity: "Normal".to_string(),
+            description: "Selectable label needs right-click copy menu".to_string(),
+        },
+        Data {
+            fixed: true,
+            number: 50939,
+            severity: "Normal".to_string(),
+            description: "Add shift clicking to GtkTextView".to_string(),
+        },
+        Data {
+            fixed: false,
+            number: 6112,
+            severity: "Normal".to_string(),
+            description: "netscape-like collapsable toolbars".to_string(),
+        },
+        Data {
+            fixed: false,
+            number: 1,
+            severity: "Normal".to_string(),
+            description: "First bug :=)".to_string(),
+        },
+    ];
+
+    let store = gtk::ListStore::new(&col_types);
+
+    let col_indices: [u32; 8] = [0, 1, 2, 3, 4, 5, 6, 7];
+
+    for (d_idx, d) in data.iter().enumerate() {
+        let icon_name = if d_idx == 1 || d_idx == 3 {
+            "battery-caution-charging-symbolic"
+        } else {
+            ""
+        };
+
+        let sensitive = d_idx != 3;
+
+        let values: [&dyn ToValue; 8] = [
+            &d.fixed.to_value(),
+            &d.number.to_value(),
+            &d.severity.to_value(),
+            &d.description.to_value(),
+            &0u32.to_value(),
+            &icon_name.to_value(),
+            &false.to_value(),
+            &sensitive.to_value(),
```

Comment:
You don't have to call `to_value()` here, just taking the reference should be enough

Reply:
And maybe this would be nicer as a method on the `Data` struct?